calendar.danish

```diff
@@ -0,0 +1,11 @@
+/*
+ * Danish calendar file(s)
+ *
+ */
+
+#ifndef _calendar_danish_
+#define _calendar_danish_
+
+#include <da_DK/calendar.all>
```

Comment:
I am sorry, there is still one reference to da_DK directory that needs to be switched to da_DK.UTF-8. Once this is fixed too, I think it is ready to be merged.

Reply:
Ooops - I'm sorry for the oversight. This is now fixed.